src/services/authService.ts

```diff
@@ -24,6 +23,11 @@ interface ConfirmEmailResponse {
   httpStatusCode: number
 }
 
+interface ErrorResponse {
+  message: string
+  httpStatusCode: number
+}
+
```

Comment:
Just for my understanding: why `src/models/ErrorResponse.ts` cannot be used instead? What is the purpose of yet another type?